src/main/java/jenkins/plugins/mattermost/StandardMattermostService.java

```diff
@@ -50,10 +54,10 @@ public boolean publish(String message, String color) {
 			// - @dmchannel
 			int atPos = userAndRoomId.indexOf("@");
 			if (atPos > 0 && atPos < userAndRoomId.length() - 1) {
-			    userId = userAndRoomId.substring(0, atPos).trim();
-			    roomId = userAndRoomId.substring(atPos + 1).trim();
+					userId = userAndRoomId.substring(0, atPos).trim();
+					roomId = userAndRoomId.substring(atPos + 1).trim();
 			}
-        
```

Comment:
looks like these were white spaces that had been inadvertently left here, so I took the opportunity to convert them to tabs.